src/fixturelibrary.ts

```diff
@@ -127,6 +127,11 @@ export class FixtureLibrary {
     return fixture;
   }
 
+  public async getIndex(): Promise<string[]> {
+    const items = await this.fixtureIndex.getIndex();
+    return Object.keys(items);
```

Comment:
This output is fine for computers but not great for humans. 

All of the fixtures and manufactures have human readable names 

Each fixture and manufacturer has a `name` property that you can take human readable names from


See the designs: 
<img width="744" alt="image" src="https://user-images.githubusercontent.com/5770007/192375134-c1807721-db9d-4aca-9083-f7005afffab3.png">


See the fixtures: https://github.com/integrated-cinematics/open-fixture-library, Schemas are here: https://github.com/integrated-cinematics/open-fixture-library/tree/master/schemas